src/main/js/helpers/axis.js

```diff
@@ -189,22 +190,38 @@ const getAxesScale = (axis, scale, config) => {
         scale.x,
         getAxisInfoOrientation(config.axis.x.orientation)
     );
+
+    if (
+        (config.ticksCount === undefined ||
+            config.ticksCount > constants.TICKSCOUNT_MAXLIMIT) &&
```

Comment:
can we add about max tick limit in documentation.

Reply:
Added [here](https://github.com/cerner/carbon-graphs/pull/216/commits/0aa856892546b52db5cf458fea5cf8a8f333fd4a). Is this what you mean?

Reply:
I mean to add it in the ticks.md file.

Reply:
It is [added](https://github.com/cerner/carbon-graphs/pull/216/commits/3a91cc57f6dcf9fdc6a4eb793e66e08fedac093b) in Ticks.md.

Reply:
It is [mentioned](https://github.com/cerner/carbon-graphs/pull/216/commits/3a91cc57f6dcf9fdc6a4eb793e66e08fedac093b) in the `ticks.md` file.

Reply:
i cannot see where you added max tick count in ticks.md file, can you please share the link of exact line.